bin/arm-template-merge

```diff
@@ -126,7 +128,14 @@ async function mergeTemplateWith(target, inFile) {
 }
 
 async function loadTemplate(inFile) {
-  let json = await readFile(inFile, 'utf8');
+  let jsonBuf = await readFile(inFile);
+
```

Comment:
Suggestion for the strip 3 first characters.
```suggestion
const json = (jsonBuf.slice(0, 3).equals(utf8BOM) ? jsonBuf.slice(3) : jsonBuf).toString('utf8');
```

Reply:
I'd rather have it more readable. Also, it's clearer where to focus should we decide to support more encodings.